### PR TITLE
use storybook-snapper

### PR DESCRIPTION
**Pull request by jonathanadler**
add storybook-snapper + use it instead of local Snapper

### CODE REVIEW

File: applitools.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const { execSync } = require('child_process');
 const appName = require('./package.json').name;
 const merge = require('lodash/merge');
-const { DATA_READY_HOOK } = require('./test/visual/dataHooks.js');
+const { DATA_READY_HOOK } = require('storybook-snapper/dist/src/index');
```
Comment (shlomitc) [inline applitools.config.js]: I'd ditch this one as discussed privately

File: tsconfig.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   },
   "include": [
     "./node_modules/wix-ui-core/dist/src/globals.d.ts",
+    "./node_modules/storybook-snapper/src/external-types.d.ts",
```
Comment (shlomitc) [inline tsconfig.json]: why is this required?

Comment (jonathanadler) [reply tsconfig.json]: Good question.
I was having some problems with importing the library.
It might not be needed. I'll check.

File: applitools.config.js (hunk 1 above)
Comment (jonathanadler) [reply applitools.config.js]: So you're saying `storybook-snapper` shouldn't have a default selector for this?

My only problem is that we need this selector both in the config file, and in the storybook  react component.
I think their design is kind of limiting in this way.

And since I really like having a single source of truth, I'll still need to import the hook from a datahook file.
